src/index.html

```diff
@@ -7,8 +7,96 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__catalog logo">
+        <img
+          href="home"
+          src="./images/logo.png"
```

Comment:
Remove the `href` attribute from the `img` tag as it is not a valid attribute for images. The correct attribute is `src`, which is already included.